template/.eslintrc.js

```diff
@@ -1,46 +0,0 @@
-module.exports = {
```

Comment:
...and this shouldn't be deleted either.

There are many other changes that don'T belong here, and I will not comment on all of them.